app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -14,33 +16,31 @@ def index
 
       def addresses(search_term)
         addresses = geocoder.find_address_candidates(SingleLine: search_term).slice(0, 1)
+
         { json: addresses, status: :ok }
-      rescue Faraday::ConnectionFailed => err
-        analytics.idv_arcgis_request_failure(
-          api_status_code: 422,
-          exception_class: err.class,
-          exception_message: err.message,
-          response_body_present: err.respond_to?(:response_body) && err.response_body.present?,
-          response_body: err.respond_to?(:response_body) && err.response_body,
-          response_status_code: err.respond_to?(:response_status) && err.response_status,
-        )
-        { json: [], status: :unprocessable_entity }
-      rescue Faraday::TimeoutError => err
-        analytics.idv_arcgis_request_failure(
-          api_status_code: 422,
-          exception_class: err.class,
-          exception_message: err.message,
-          response_body_present: err.respond_to?(:response_body) && err.response_body.present?,
-          response_body: err.respond_to?(:response_body) && err.response_body,
-          response_status_code: err.respond_to?(:response_status) && err.response_status,
-        )
-        { json: [], status: :unprocessable_entity }
       end
 
       def geocoder
         @geocoder ||= IdentityConfig.store.arcgis_mock_fallback ?
           ArcgisApi::Mock::Geocoder.new : ArcgisApi::Geocoder.new
       end
+
+      def report_errors(error)
+        remapped_error = {
+          Faraday::ConnectionFailed => :unprocessable_entity,
+          Faraday::TimeoutError => :unprocessable_entity,
+        }[error.class] || :internal_server_error
+
+        analytics.idv_in_person_locations_request_failure(
```

Comment:
What was the motivation for changing the analytics method from `idv_arcgis_request_failure` to `idv_in_person_locations_request_failure`?  

Since this is the address search controller, and `idv_arcgis_request_failure` is documented as `Tracks if request to get address candidates from ArcGIS fails`, I think this should stay as `idv_arcgis_request_failure`.  

Let me know if I'm missing something.

Reply:
Also, it might be helpful to add an expect statement to `address_search_controller_spec.rb` that tests if the analytics method is called when there's an error.